### DISCUSSION TITLE
Why is remapping Escape not working?

### DISCUSSION THREAD

**Discussion by LingLambda** [hyprwm/Hyprland#8548] [Q&A] [answered]
![图片](https://github.com/user-attachments/assets/f039db18-cb55-4cfe-83c2-f94d69fbf2a3)

my config
![图片](https://github.com/user-attachments/assets/acbd314e-6b16-406f-b1f4-a8c3d0b6ffcd)

`ctrl:nocaps` works, but I want to remap Escape.

**Comment by MahouShoujoMivutilde** [comment, marked as answer]
Do you want to swap caps with escape?

There are

* `caps:swapescape` - Swap Esc and Caps Lock
* `caps:escape` - Make Caps Lock an additional Esc
* `caps:escape_shifted_capslock` - Make Caps Lock an additional Esc, but Shift + Caps Lock is the regular Caps Lock

from `man xkeyboard-config`
